crates/driver/src/infra/config/file/mod.rs

```diff
@@ -63,10 +63,19 @@ struct Config {
     )]
     order_priority_strategies: Vec<OrderPriorityStrategy>,
 
+    /// The maximum number of `/settle` requests that can be queued up
+    /// before the driver starts dropping new requests.
```

Comment:
Instead of still participating in the auction and then later dropping `/settle` requests a full queue should probably result in the driver returning errors on `/solve` requests instead. Otherwise the solver could rake up slashings for winning auctions and not settling them later on.

Reply:
Totally makes sense. Added this.